app/application/usecases/course/detail_usecase.py

```
@@ -1,9 +1,64 @@
+        self,
+        scraper: BaseLmsScraper,
+        domain_service: CourseService,
+    ) -> None:
         self.scraper = scraper
+        self.domain_service = domain_service
+
+    async def get_course_detail(
+        self,
+        course_code: str,
+        course: Optional[CourseEntity] = None,
+        raise_exception: bool = True,
```

Ruff (0.15.14)

[warning] Boolean-typed positional argument in function definition

(FBT001)